modules/client-configuration/pages/vhm-aws.adoc

```diff
@@ -38,5 +38,5 @@ To see which objects and resources have been inventoried, navigate to menu:Syste
 Instances running on the Amazon public cloud report a UUID to the {productname} Server in the format of an ``i`` followed by seventeen hexadecimal digits:
 
 ----
-i1234567890abcdef0
+i-1234567890abcdef0
```

Comment:
Looks like this might need to be an uppercase "I", not lowercase?

Reply:
That's right: in SUMA, it is shown in uppercase.

Reply:
Fixed